### ISSUE TITLE
Sort eMASS Test Results Report Spreadsheet Output

### ISSUE THREAD

**Issue by TheCleverEpithet** [squinky86/STIGQter#47]
If possible, can the "eMASS Test Results" Report Spreadsheet output be sorted by RMF Control (Column A - "Control Acronym") by default?  The first few rows in the template format may make this difficult to do (as the controls/data do not start until around row 7) and if it is not feasible, we can do it manually in Excel each time the report is created. Low Priority.

**Comment by squinky86** [comment]
I think it's as fixed as its going to get. The changes are:

- The rows are in sorted order by the CCI column
- The filters are added automatically to the report in order to do any further sorting you may need

Wait for tonight's nightly - the one from this morning has a very broken eMASS test result report.
